### PR TITLE
pre-commit: test PR80607

### PR DESCRIPTION
**Pull request by dtcxzyw**
PR Link: https://github.com/llvm/llvm-project/pull/80607

### CODE REVIEW

File: bench/assimp/optimized/glTFExporter.cpp.ll [hunk 1]
```diff
@@ -67619,7 +67617,7 @@ if.then.i.i233:                                   ; preds = %.noexc253, %.noexc2
 .noexc255:                                        ; preds = %if.then.i.i233
   %sub.i.i237 = sub i32 %symbol.addr.011.i.i235, %shl12.i.i234
   %inc.i.i238 = add nuw nsw i32 %k.addr.010.i.i236, 1
-  %shl.i.i239 = shl i32 2, %k.addr.010.i.i236
+  %shl.i.i239 = shl nuw i32 2, %k.addr.010.i.i236
```
Comment (arsenm) [inline bench/assimp/optimized/glTFExporter.cpp.ll:67620]: Don't understand what I'm looking at in this patch; this looks unrelated?

Comment (dtcxzyw) [reply bench/assimp/optimized/glTFExporter.cpp.ll:67620]: The result is deterministic. I believe this change is related to the patch.

Comment (arsenm) [reply bench/assimp/optimized/glTFExporter.cpp.ll:67620]: Oh, I was largely confused because this isn't an llvm test at all and is your separate repo
